app/gui/src/project-view/components/GraphEditor/widgets/WidgetIcon.vue

```diff
@@ -39,10 +39,11 @@ export const widgetDefinition = defineWidget(
   <div :class="{ WidgetIcon: true, noGap: props.input[DisplayIcon].noGap === true }">
     <div class="iconContainer">
       <Transition>
-        <LoadingSpinner
+        <GrowingSpinner
```

Comment:
Please keep the prop names the same

Reply:
It was requested by Kaz in https://github.com/enso-org/enso/pull/13070#discussion_r2081803515 and the change makes sense, doesn't it?

Reply:
Then lets rename it in react as well